### PR TITLE
showUsers squashed commit includes:

### PR DESCRIPTION
**Pull request by Lantrie**
showUsers third commit
showUsers second commit
showUsers first commit

### CODE REVIEW

File: src/User.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+public class User {
+    private int id;
+    private String name;
```
Comment (blackspyek) [inline src/User.java:3]: change eto username

**Review by blackspyek — CHANGES_REQUESTED** [review]
Just change name

File: src/User.java (hunk 1 above)
Comment (Lantrie) [reply src/User.java]: Done!

**Review by blackspyek — APPROVED** [review]
lgtm

**Comment by blackspyek** [comment]
LGTM

**Pull request merged**